packages/node/src/__tests__/graceful-shutdown-integration.test.ts

```diff
@@ -30,14 +34,14 @@ describe('Ability for users to exit without losing events', () => {
   }
 
   describe('drained emitted event', () => {
-    test('emits a drained event if only one event is dispatched', async () => {
+    it('emits a drained event if only one event is dispatched', async () => {
```

Comment:
I changed `test` to `it` just because my IDE thought `test()` had to return a promise, so I had tons of red squiggles. Might be a transient type merge occurring, but `it` matches what we use in out other packages and fixed my issue :grin: